libs/map/framework.cpp

```diff
@@ -854,7 +854,7 @@ void Framework::ShowBookmark(Bookmark const * mark)
   es.SetIsVisible(mark->GetGroupId(), true /* visible */);
 
   if (m_drapeEngine)
-    m_drapeEngine->SetModelViewCenter(mark->GetPivot(), scale, true /* isAnim */, true /* trackVisibleViewport */);
+    m_drapeEngine->SetModelViewCenter(mark->GetPivot(), scale, true /* isAnim */, true /* ignoreVisibleViewport */);
```

Comment:
But, I don't understand why we have ```ignoreVisibleViewport == true``` here and below, after changing the param semantics.